oat-sa/extension-tao-itemqti
# Feature/ms 131/support longinterpretation attribute

Ticket: https://oat-sa.atlassian.net/browse/MS-131

**What has changed?**

* Long interpretation attribute is added
* Validation for the attribute is added (accepts valid URI)
* Tests are added

**How to test?**

* Create an item
* Add an interaction
* Go to response tab
* in the right sidebar, add an outcome declaration
* You should be able to see a new attribute by the name Long Interpretation
* Add a valid URI
* Save the item and export
* The item XML should have the long interpretation attribute present with the value entered by you 

**Run unit tests**

`cd tao/views/build`

`npx grunt connect:test qunit:single --test=/taoQtiItem/views/js/test/qtiCreator/plugins/panel/outcomeEditor/test.html`
